MekHQ/src/mekhq/AtBGameThread.java

```diff
@@ -56,18 +57,29 @@ public class AtBGameThread extends GameThread {
     private final AtBScenario scenario;
     private final BehaviorSettings autoResolveBehaviorSettings;
 
+    /**
+     * Constructor for AtBGameThread
+     *
+     * <p>
+     *     This constructor creates a new AtBGameThread with the given name, password, client, MekHQ application, list of
+     *     units, scenario, and auto resolve behavior settings. The game thread is started by default.
+     * </p>
+     *
+     * @param name                        The name of the player
+     * @param password                    The password for the game
+     * @param c                           The client
+     * @param app                         The MekHQ application
+     * @param units                       The list of units to import into the game
+     * @param scenario                    The scenario to use for this game
+     * @param autoResolveBehaviorSettings The behavior settings for the auto resolve bot
+     */
     public AtBGameThread(String name, String password, Client c, MekHQ app, List<Unit> units,
```

Comment:
Very minor point. Can we change the param `c` to `client`. We're trying to move away from single-letter and abbreviated param names as they cause a lot of confusion. You don't need to go crazy, just if you happen to notice any in a method you're changing it's a good habit to get into.